client/src/pages/buildPage/buildPage.tsx

```diff
@@ -15,15 +15,24 @@ import { getBuild } from 'store/builds/getBuild';
 import { clearState } from 'store/builds/clearState';
 import { runBuild } from 'store/builds/runBuild';
 
+import { State } from 'store/store';
+import { BuildsState } from 'store/builds/reducer';
+
 import { getBuildStatus } from 'utils/build';
 
 import './buildPage.css';
 
-const BuildPage = props => {
-  const { id } = useParams();
+export interface BuildPageProps {
+  history: MemoryHistory;
+}
+
+const BuildPage: React.FC<BuildPageProps & RouteComponentProps> = props => {
+  const { id = '' } = useParams();
   const dispatch = useDispatch();
-  const { build, loading, logLoading } = useSelector(state => state.builds);
-  const repoName = useSelector(state => state.settings.settings?.repoName || 'Build details');
+  const { build, loading, logLoading } = useSelector((state: State): BuildsState => state.builds);
```

Comment:
если опишешь возвращаемый тип редьюсера
```suggestion
  const { build, loading, logLoading } = useSelector((state: State) => state.builds);
```

Reply:
Так и сделала, спасибо!